packages/markup-frontstage/src/util/MarkupTypes.ts

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { MarkupUiEvent } from "./MarkupUiEvents";
+import { SavedViewData } from "./SavedViewTypes";
+
+/**
+ * Markup frontstage constants.
+ */
+export enum MarkupFrontstageConstants {
+  DRAWING_TOOLS = "DrawingTools",
+  FRONTSTAGE_ID = "BentleyMarkupFrontstage",
+  TOOL_SETTINGS_WIDGET_KEY = "MarkupToolSettingsWidget",
+  TOOL_WIDGET_KEY = "MarkupToolWidget",
+  VIEW_LAYOUT_ID = "MarkupView",
+  WIDGET_ID = "MarkupWidget",
+  WIDGET_KEY = "MarkupWidget",
+}
+
+/**
+ * Mapping of view element and nine-zone layout position in the viewport
+ * e.g.
+ * {
+      topCenter: (
+        <Zone
+          widgets={[
+            <Widget
+              key={MarkupFrontstageConstants.TOOL_SETTINGS_WIDGET_KEY}
+              isToolSettings={true}
+            />,
+          ]}
+        />
+      )
+    }
+ */
```

Comment:
Are you leaving this commented out code snippet for a reason?

Reply:
Yes sir, it's the example for what is expected in ViewElementDictionary :)